source/uwp/SharedRenderer/lib/XamlBuilder.cpp

```diff
@@ -213,7 +213,13 @@ namespace AdaptiveCards::Rendering::Xaml_Rendering
             bodyElementHostImpl->SetAdaptiveHeight(true);
             winrt::WholeItemsPanel bodyElementHost = *bodyElementHostImpl;
 
-            XamlHelpers::ApplyMarginToXamlElement(hostConfig, bodyElementHost);
+            // Override the root styles
+            XamlHelpers::SetStyleFromResourceDictionary(renderContext, L"Adaptive.RootContainer", bodyElementHost);
```

Comment:
I think we should keep the name centered around elements, other styles have names as Adaptive.Column, I think in this case it could be Adaptive.Card or something similar

@paulcam206 , @jwoo-msft , what do you think?

Reply:
It looks like you already have such a style used by BuildXamlTreeFromAdaptiveCard however because ApplyMarginToXamlElement is called which calls .Margin() directly, the style override of Margin won't work.

Reply:
I updated the approach so that a new style name is not required.